### PR TITLE
Check if binary tree symmetric or not issue - #2880

### PR DESCRIPTION
**Pull request by simran130810**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

This is java implementation of checking if binary tree is Symmteric or not.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This is solution for issue-2880.

### CODE REVIEW

File: Java/ds/SymmetricBT.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    }
+    
+}
+
+//input - 
+//example 1 : [3,9,20,-1,-1,15,7]
+//output - if tree is symmetric :- true  
+//         else :- false
+//Time Complexity :- O(N)
+//Space Complexity :- O(1)
```
Comment (HarshCasper) [inline Java/ds/SymmetricBT.java:101]: Use Multi-Line Comments

File: Java/ds/SymmetricBT.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
+//this program checks if a binary tree is symmetric or not through checking
+//if a binary tree has mirror image of itself with O(n) time complexity
```
Comment (HarshCasper) [inline Java/ds/SymmetricBT.java:2]: Use Multi-Line Comments and proper grammar as well as sentence formation

File: Java/README.md [hunk 3]
```diff
@@ -133,9 +133,11 @@ _add list here_
 - [Trie Insert and Search String](ds/Trie_with_Hashmap.java)
 - [Priority Queue](ds/PriorityQueueEg.java)
 - [AVL Tree](ds/AVL_Tree.java)
+- [Check if a binary tree is symmetric or not](ds/SymmetricBT.java)
 - [Balanced Brackets](ds/balancedBrackets.java)
 - [Remove Duplicate in a Unsorted Linked List without extra Space](ds/RemDupInLLWithoutExtraSpace.java)
 
+
```
Comment (HarshCasper) [inline Java/README.md:140]: Remove this extra line

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Format your code properly

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this PR due to inactivity.

**Pull request closed without merging**